README.md

```diff
@@ -1,44 +1,88 @@
+`RAILS_ENV=production`.
 
-## Setup
+## Deployment
 
-### Prerequisites
+There currently are two hosted environments in AKS, `development` and
+`production`, and the respective urls are:
 
-This project depends on:
+- https://development.calculate-teacher-pay.education.gov.uk/
+- https://calculate-teacher-pay.education.gov.uk/
 
-  - [Ruby](https://www.ruby-lang.org/)
-  - [Ruby on Rails](https://rubyonrails.org/)
-  - [NodeJS](https://nodejs.org/)
-  - [Yarn](https://yarnpkg.com/)
-  - [Postgres](https://www.postgresql.org/)
+Every merge to `main` will trigger a deployment to `development` and
+`production` via a Github Action workflow.
 
-### Linting
+In some circumstances, you may need to trigger a manual deployment to
+`development` first, e.g. to test a new feature before it's ready for release.
 
-To run the linters:
+To trigger a manual deployment, you can use the Github Action workflow
+"Deploy to AKS (Manual)", specifying a target docker image tag and environment.
 
-```bash
-bin/lint
+For `development`, you'd normally need to build the docker image locally and
+push it to the repo's docker registry. Example steps:
+
+```
+$ export LATEST_COMMIT_SHA=git rev-parse --verify HEAD
+$ docker buildx build --platform linux/amd64 --output type=docker . \
+  -t ghcr.io/dfe-digital/teacher-pay-calculator:$LATEST_COMMIT_SHA
+$ docker push ghcr.io/dfe-digital/teacher-pay-calculator:$LATEST_COMMIT_SHA
 ```
+
+To authenticate with the registry, make sure you are added to it and
+follow the instructions at https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic
+
+** **CAVEATS** **
+
+* You can use the same workflow to deploy to `production`, although you should
+  use the deployment via pipeline approach; in other words, simply merge to
+  `main` when a branch/PR is ready to go to `production`
+* The current setup relies on a Personal Github Token created by one of the
+  current maintainers of the project and added to the `s189t01-ctp-dv-inf-kv`
+  and `s189t01-ctp-pd-inf-kv` keyvaults; this is not ideal, but it was necessary
+  to keep the repository public while we were doing the first and initial
+  testing of new salary figures that couldn't be disclosed to the public until
+  an official government announcement
+* The Personal Github Token should have an expiration date and refreshed before
+  completely expiring, otherwise the currently deployed pods won't be able to
+  restart and fetch the docker image, potentially leading to unplanned downtime
+* The `development` environment is protected with Basic auth; the credentials
+  can be found in the `s189t01-ctp-dv-inf-kv` keyvault.
+
+## Monitoring
+
+Sentry is configured for both hosted environments.
+
+StatusCake is also configured to check against the `/healthcheck` endpoint.
+
+Alerts will be sent to the #claim_tech Slack channel.
```

Comment:
as well as email and phone, provided people are added to the [Teacher payments contact group](https://app.statuscake.com/ContactGroup.php?CUID=195955)

Reply:
Addressed with [3bfa7ff](https://github.com/DFE-Digital/teacher-pay-calculator/pull/63/commits/3bfa7ff8f96a433c680a54bf2cc9fd9643a3e1df)